.github/workflows/Test.yml

```diff
@@ -63,7 +63,7 @@ jobs:
             fail-fast: false
             matrix:
                 os: [ 'ubuntu-latest' ]
-                typo3: [ '^11.5.26', '^12.4' ]
+                typo3: [ '^11.5.26', '^12.4', '^13.1' ]
```

Comment:
This won't work, you'll need to add TYPO3v13 via a dedicated `include` entry for PHP 8.2 and 8.3 instead.

Reply:
Excluding 8.1 would be an option, too.
```yaml
exclude:
  - typo3: "^13.1"
    php: "8.1"
```